modules/reactor/src/meshgenerators/PatternedHexMeshGenerator.C

```diff
@@ -861,6 +868,32 @@ PatternedHexMeshGenerator::generate()
         out_mesh->add_point(p_tmp, node_azi_list[i * side_intervals + j - 1].second);
       }
     }
+
+    // adjust quadratic mid-edge nodes after boundary node redistribution
```

Comment:
I think this can be a method that can be called by both Cart and Hex.

Reply:
extracted to common base class